### ISSUE TITLE
Enchanted mentions

### ISSUE THREAD

**Issue by AZsSPC** [Wiicart/Strings#6]
I see levitsky's issue about mentions, but what about modify it?
I'm talking about mentioning groups of players, commands and other, it can be helpful for helping to newbies:
- i have a problem, can anyone help?
- just click on the `@admin` and text to them
***
- `@vip's`, come here
***
- just press on `@[example command]` and it pastes in your input bar

hope you'll get the idea

**Comment by Wiicart** [comment]
Yeah, sure.  Can you explain what you mean about commands though?

**Comment by AZsSPC** [comment]
sure!
im minecraft exist functionality of tellraw commands for this:
{"text":"click for magic","clickEvent":{"action":"suggest_command","value":"say maaagic"}}

and all thing to do is search through players input like mention system, but now for template `@[*any text there*]` and give this label an onclickEvent with suggestiong this comand to player

also it helpfull not only for command but text too, for example `@[!to global chat example]` can aesily make newbies undertand chat system
